docroot/sites/emergency.uiowa.edu/modules/emergency_core/emergency_core.module

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * @file
+ * Custom functionality the Emergency website.
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\emergency_core\AlertsProcessor;
+
+/**
+ * Implements hook_form_alter().
+ */
+function emergency_core_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  switch ($form_id) {
+    case 'node_hawk_alert_form':
+    case 'node_hawk_alert_edit_form':
+      // To prevent duplicate alerts, identifier field is not editable.
+      $form['field_hawk_alert_identifier']['#disabled'] = TRUE;
+      $form['field_hawk_alert_identifier']['#markup'] = t('This field is managed through an automated process and cannot be changed.');
+      break;
+  }
+}
+
+/**
+ * Import alerts.
+ */
+function emergency_core_import_alerts() {
+  // Check that this process is being called with
+  // admin permissions.
+  /** @var Drupal\uiowa_core\Access\UiowaCoreAccess $check */
+  $check = \Drupal::service('uiowa_core.access_checker');
+  /** @var Drupal\Core\Access\AccessResultInterface $access */
+  $access = $check->access(\Drupal::currentUser()->getAccount());
+  if (!$access) {
+    return 'This is a restricted process.';
+  }
+
+  $logger = \Drupal::logger('emergency_core');
+  $logger->notice('Starting the hawk alert content sync.');
+
+  $sync_service = new AlertsProcessor();
+  $sync_service->init();
+  $sync_service->process();
+
+  $arguments = [
+    '@created' => $sync_service->getCreated(),
+    '@updated' => $sync_service->getUpdated(),
```

Comment:
These don't get updated/deleted right? Just created?

Reply:
Correct! Only created and skipped ever change. I originally left the others in to test I had completely removed everything. But perhaps I should remove them now to avoid confusion.